### PR TITLE
Reporting updates

### PR DESCRIPTION
**Pull request by N-lson**
Mainly updating the section on `DeploymentHistory` update frequency. Also includes some small improvements elsewhere.

### CODE REVIEW

File: src/pages/docs/administration/reporting/report-on-deployments-using-excel.md [hunk 1]
```diff
@@ -7,11 +7,7 @@ description: How to report on deployments using Excel & XML
 navOrder: 10
 ---
 
-:::div{.hint}
-Also take a look at the [Reporting features](/docs/administration/reporting) which provides reporting optimized views of the live deployment data, no longer requiring an XML export.
-:::
```
Comment (N-lson) [inline src/pages/docs/administration/reporting/report-on-deployments-using-excel.md:12]: I removed this hint as I found it confusing. It just takes you back to the reporting index and the description doesn't really mean anything. I guess it's suggesting the database approach? You can always just go straight to the database page anyway.

File: src/pages/docs/administration/reporting/report-on-deployments-with-sql.md [hunk 2]
```diff
@@ -29,9 +28,7 @@ The table is completely denormalized, and should have any information that you m
 
 ## How often is the data updated?
 
-The data in the table (and exposed by the feed) updates every 30 seconds to add deployments that have recently completed.
-
-Since the data is denormalized, changing the name of a project or environment, or changing the version number of a release, may result in stale data. Octopus corrects this every 24 hours on a schedule.
+The data in the table (and exposed by the feed) updates every time any data related to deployments changes. This includes changes such as changing the name of a project or environment, or changing the version number of a release. The data should always be up-to-date within the last 30 seconds, however if Octopus is performing many operations this may increase to several minutes.
```
Comment (N-lson) [inline src/pages/docs/administration/reporting/report-on-deployments-with-sql.md:31]: I arbitrarily picked these timings (30 seconds to match the old 30 seconds). Happy to change them if they don't feel accurate.

File: src/pages/docs/administration/reporting/report-on-deployments-using-excel.md [hunk 3]
```diff
@@ -7,11 +7,7 @@ description: How to report on deployments using Excel & XML
 navOrder: 10
 ---
 
-:::div{.hint}
```
Comment (felixi-octopus) [inline src/pages/docs/administration/reporting/report-on-deployments-using-excel.md:10]: Scope creep: Do we need to adjust the title? It looks like it's being escaped or something
image

File: src/pages/docs/administration/reporting/report-on-deployments-with-sql.md (hunk 2 above)
Comment (felixi-octopus) [reply src/pages/docs/administration/reporting/report-on-deployments-with-sql.md:31]: Nit: Or not mention it?
```suggestion
The data in the table (and exposed by the feed) updates every time any data related to deployments changes. This includes changes such as changing the name of a project or environment, or changing the version number of a release. The data should always be up-to-date, however if Octopus is performing many operations, data could be stale up to several minutes.
```

**Review by felixi-octopus — APPROVED** [review]
LGTM. Minor nits

File: src/pages/docs/administration/reporting/report-on-deployments-using-excel.md (hunk 3 above)
Comment (N-lson) [reply src/pages/docs/administration/reporting/report-on-deployments-using-excel.md:10]: Changed to "and"
image
